MouseAimFlight/MouseAimVesselModule.cs

```diff
@@ -311,5 +315,16 @@ void OnDestroy()
             if (vessel)
                 vessel.OnAutopilotUpdate -= MouseAimPilot;
         }
+
+        static T Clamp<T>(T value, T min, T max)
+        where T : System.IComparable<T>
```

Comment:
What is this?

Reply:
I'm a absolute idiot and didn't saw that unity api has a built in clamp function and just pasted some code from some website..